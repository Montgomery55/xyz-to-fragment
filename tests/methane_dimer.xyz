10

  C   -1.5156709   -0.0102737   -0.3057180
  C    5.0886808    0.4289014   -7.4914331
  H   -0.4062707   -0.0104501   -0.3065486
  H   -1.8859260    0.7092265   -1.0646655
  H   -1.8846743    0.2875722    0.6972255
  H   -1.8858165   -1.0274430   -0.5488820
  H    6.1980815    0.4292265   -7.4906283
  H    4.7192327    1.1385016   -8.2600330
  H    4.7180632    0.7393315   -6.4929110
  H    4.7193479   -0.5914547   -7.7221638
