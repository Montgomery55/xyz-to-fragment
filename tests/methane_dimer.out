10
1    C         -1.51567090  -0.01027370  -0.30571800	64    	3 4 5 6
2    C          5.08868080   0.42890140  -7.49143310	64    	7 8 9 10
3    H         -0.40627070  -0.01045010  -0.30654860	11    	1
4    H         -1.88592600   0.70922650  -1.06466550	11    	1
5    H         -1.88467430   0.28757220   0.69722550	11    	1
6    H         -1.88581650  -1.02744300  -0.54888200	11    	1
7    H          6.19808150   0.42922650  -7.49062830	11    	2
8    H          4.71923270   1.13850160  -8.26003300	11    	2
9    H          4.71806320   0.73933150  -6.49291100	11    	2
10   H          4.71934790  -0.59145470  -7.72216380	11    	2
